Mathlib/Analysis/Complex/UpperHalfPlane/FunctionsBoundedAtInfty.lean

```diff
@@ -53,18 +53,35 @@ def zeroAtImInftySubmodule (α : Type*) [NormedField α] : Submodule α (ℍ →
 def boundedAtImInftySubalgebra (α : Type*) [NormedField α] : Subalgebra α (ℍ → α) :=
   boundedFilterSubalgebra _ atImInfty
 
-nonrec theorem IsBoundedAtImInfty.mul {f g : ℍ → ℂ} (hf : IsBoundedAtImInfty f)
+namespace IsBoundedAtImInfty
+
+variable {α : Type*} {f g : ℍ → α}
+
+nonrec theorem add [SeminormedAddCommGroup α] (hf : IsBoundedAtImInfty f)
+    (hg : IsBoundedAtImInfty g) : IsBoundedAtImInfty (f + g) := by
+  simpa only using hf.add hg
+
+nonrec theorem sub [SeminormedAddCommGroup α] (hf : IsBoundedAtImInfty f)
+    (hg : IsBoundedAtImInfty g) : IsBoundedAtImInfty (f - g) := by
+  simpa only using hf.sub hg
+
+nonrec theorem mul [SeminormedRing α] (hf : IsBoundedAtImInfty f)
     (hg : IsBoundedAtImInfty g) : IsBoundedAtImInfty (f * g) := by
-  simpa only [Pi.one_apply, mul_one, norm_eq_abs] using hf.mul hg
+  simpa only using hf.mul hg
+
+end IsBoundedAtImInfty
 
-theorem bounded_mem (f : ℍ → ℂ) :
-    IsBoundedAtImInfty f ↔ ∃ M A : ℝ, ∀ z : ℍ, A ≤ im z → abs (f z) ≤ M := by
+theorem isBoundedAtImInfty_iff {α : Type*} [Norm α] {f : ℍ → α} :
+    IsBoundedAtImInfty f ↔ ∃ M A : ℝ, ∀ z : ℍ, A ≤ im z → ‖f z‖ ≤ M := by
   simp [IsBoundedAtImInfty, BoundedAtFilter, Asymptotics.isBigO_iff, Filter.Eventually,
     atImInfty_mem]
 
-theorem zero_at_im_infty (f : ℍ → ℂ) :
```

Comment:
this (and the others) looks like an old lean 3 name that got forgotten in the port.